linebot/webhook_test.go

```diff
@@ -1092,3 +1229,28 @@ func BenchmarkParseRequest(b *testing.B) {
 		client.ParseRequest(req)
 	}
 }
+
+func BenchmarkGetWebhookInfo(b *testing.B) {
+	server := httptest.NewTLSServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		defer r.Body.Close()
+		w.Write([]byte(`{"userId":"U","basicId":"@B","premiumId":"P","displayName":"BotTest","pictureUrl":"https://","chatMode":"chat","markAsReadMode":"manual"}`))
```

Comment:
for GetWebhookInfo() the response should be "{
  "endpoint": "https://example.herokuapp.com/test",
  "active": "true"
}"

Reply:
Sorry, I forget change it. So fixed it.